src/PgSqlHandle.php

```diff
@@ -265,6 +266,7 @@ private function createResult($result)
                 foreach (self::DIAGNOSTIC_CODES as $fieldCode => $desciption) {
                     $diagnostics[$desciption] = \pg_result_error_field($result, $fieldCode);
                 }
+                $diagnostics['sql'] = $sql;
```

Comment:
What's the reason for adding this?

Reply:
$diagnostics will be used in QueryExecutionError for error message. 'sql' key will contain failed query